### PR TITLE
Edit styles and responsive design

### PR DESCRIPTION
**Pull request by umarsalah**
Relates #108

### CODE REVIEW

File: server/controllers/auth/login/get-user-id.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = async (req, res, next) => {
       const value = await verifyToken(token, SECRET_KEY);
       res.json({ id: value.userId });
     } else {
-      res.status(401).json({ message: 'You are not registred yet' });
+      res.json({ id: 0 });
```
Comment (ahmadfarid-059) [inline server/controllers/auth/login/get-user-id.js:17]: why to send status of 200 while the user is sending a bad request

**Review by ahmadfarid-059 — APPROVED** [review]

File: client/src/pages/Register/Login/index.jsx [hunk 2]
```diff
@@ -82,9 +82,6 @@ const Login = () => {
           requiered />
         <Button type='submit' variant="contained" sx={btnStyle}>Log in</Button>
       </form>
-      <FormHelperText id="helper-text" sx={{ color: '#fff' }}>
-          You do not have account? <Link to='/signup'> Sign Up </Link>
-      </FormHelperText>
```
Comment (ahmadfarid-059) [inline client/src/pages/Register/Login/index.jsx:87]: why removing this ?

**Review by Noorin99 — APPROVED** [review]

File: server/controllers/auth/login/get-user-id.js (hunk 1 above)
Comment (umarsalah) [reply server/controllers/auth/login/get-user-id.js:17]: this is not a request from the user

File: client/src/pages/Register/Login/index.jsx (hunk 2 above)
Comment (umarsalah) [reply client/src/pages/Register/Login/index.jsx:87]: cause the is nor route called `/signup`

**Review by ahmadfarid-059 — APPROVED** [review]

**Pull request merged**